src/components/biggive-form-field-select/biggive-form-field-select.tsx

```diff
@@ -99,4 +90,19 @@ export class BiggiveFormFieldSelect {
       </div>
     );
   }
+
+  private formatOptions(options: string | Array<{ label: string; value: string }>): {
+    label: string;
+    value: string;
+  }[] {
+    if (typeof options === 'string') {
+      options = JSON.parse(options);
```

Comment:
might it be worth adding back the ` as unknown` that we had after JSON.parse before? Without it we get `any` so we're not doing any check for whether it's really matching the type we need it to match. Not a blocker since if it's unlikely to fail in an unpredictable way in prod, but would be good to see if we can do without the `any` type.

Reply:
It does look like the code here will do the same as the old code in cases where the old code didn't error.

Reply:
hmm so even if I split the variable and add back the `as unknown` an `any` sneaks in the form of `any[]`:

![image](https://github.com/user-attachments/assets/e6b66f3a-ecb4-4578-b2c3-a5fc7bda165c)